lib/measured/measurable.rb

```diff
@@ -15,7 +16,11 @@ def initialize(value, unit)
     when BigDecimal
       value
     else
-      BigDecimal(value)
+      if value.blank?
```

Comment:
This is for `""`. Otherwise it would coerce to zero. Not clear:

```ruby
irb(main):002:0> BigDecimal.new("")
=> #<BigDecimal:7f9ce3369480,'0.0',9(9)>
```